skyportal/plot.py

```diff
@@ -783,12 +783,8 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, height=300):
     plot.add_tools(hover)
     model_dict = {}
     for i, (key, df) in enumerate(split):
-        model_dict['s' + str(i)] = plot.step(
-            x='wavelength',
-            y='flux',
-            color=color_map[key],
-            source=ColumnDataSource(df),
-            mode="center",
+        model_dict['s' + str(i)] = plot.line(
+            x='wavelength', y='flux', color=color_map[key], source=ColumnDataSource(df),
```

Comment:
we need `step` to properly show the spectra, otherwise they end up looking weird.

Reply:
this was requested in https://github.com/fritz-marshal/fritz/issues/111

Reply:
See latest commits that keep step plots.